### ISSUE TITLE
Error while retrieving data data from a variable 

### ISSUE THREAD

**Issue by lmar76** [MAVENSDC/cdflib#20]
The following steps have been executed:

1. Download file:

ftp://swarm-diss.eo.esa.int/Level1b/Entire_mission_data/MAGx_LR/Sat_A/SW_OPER_MAGA_LR_1B_20180816T000000_20180816T235959_0408.CDF.ZIP

2. Unzip downloaded file SW_OPER_MAGA_LR_1B_20180816T000000_20180816T235959_0408.CDF.ZIP

3. Read the file SW_OPER_MAGA_LR_1B_20180816T000000_20180816T235959_0408_MDR_MAG_LR.cdf contained in the ZIP archive:
```python
import cdflib
cdf = cdflib.CDF('SW_OPER_MAGA_LR_1B_20180816T000000_20180816T235959_0408_MDR_MAG_LR.cdf')
```

4. Get data from the `SyncStatus` variable:
```python
x = cdf.varget('SyncStatus')
```

At step 4, the following error is traced:
```
In [6]: cdf.varget('SyncStatus')
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-6-8887b529ae74> in <module>()
----> 1 cdf.varget('SyncStatus')

~/.local/opt/anaconda3/envs/swarmdata/lib/python3.6/site-packages/cdflib/cdfread.py in varget(self, variable, epoch, starttime, endtime, startrec, endrec, record_range_only, inq, expand, to_np)
    623             return self._read_vardata(vdr_info, epoch=epoch, starttime=starttime, endtime=endtime,
    624                                       startrec=startrec, endrec=endrec, record_range_only=record_range_only,
--> 625                                       expand=expand, to_np=to_np)
    626 
    627     def epochrange(self, epoch = None, starttime = None, endtime = None):

~/.local/opt/anaconda3/envs/swarmdata/lib/python3.6/site-packages/cdflib/cdfread.py in _read_vardata(self, vdr_info, epoch, starttime, endtime, startrec, endrec, record_range_only, expand, to_np)
   2021 
   2022         data = self._read_vvrs(vdr_info, vvr_offsets, vvr_start, vvr_end,
-> 2023                                startrec, endrec, to_np=to_np)            
   2024         if record_range_only:
   2025             return [startrec, endrec]

~/.local/opt/anaconda3/envs/swarmdata/lib/python3.6/site-packages/cdflib/cdfread.py in _read_vvrs(self, vdr_dict, vvr_offs, vvr_start, vvr_end, startrec, endrec, to_np)
   1684             else:
   1685                 filled_data = CDF._convert_np_data(\
-> 1686                                     self._default_pad(vdr_dict['data_type']),\
   1687                                     vdr_dict['data_type'],\
   1688                                     vdr_dict['num_elements'])

TypeError: _default_pad() missing 1 required positional argument: 'num_elms'
```

**Comment by bryan-harter** [comment]
I believe the latest change pushed to master fixes this problem, but let me know if you're still running into the issue!

**Comment by scivision** [comment]
Yes this is fixed in #22 
```python
import cdflib
h=cdflib.CDF('SW_OPER_MAGA_LR_1B_20180816T000000_20180816T235959_0408_MDR_MAG_LR.cdf')

x = h['SyncStatus']
>>> array([0, 0, 0, ..., 0, 0, 0], dtype=uint16)
x.shape
>>> (86400,)

t = h['Timestamp']
>>>
array([6.37015968e+13, 6.37015968e+13, 6.37015968e+13, ...,
       6.37016832e+13, 6.37016832e+13, 6.37016832e+13])

```

**Comment by lmar76** [comment]
It works! Thank you!
